### PR TITLE
Migrate from django-swagger to drf-yasg

### PR DESCRIPTION
**Pull request by maurizi**
## Overview

Updates the generated Swagger API docs to use `drf-yasg`

Connects #1907 

## Demo

localhost_8081_api_docs_ (1)

## Testing Instructions

* `scripts/update`
* Verify that the output of http://localhost:8081/api/docs/ looks correct

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines
- [x] This PR is targeted at the correct branch (`develop` vs. `ogr/develop`)
- ~[ ] If this PR applies to both OAR and OGR a companion PR has been created~

### CODE REVIEW

**Comment by jwalgran** [comment]
When I browse the forms all of the fields are disabled/read-only. I tested in Firefox and Chrome and it appears to be a problem on all of the endpoints. Do you also see the fields as disabled?

**Review by jwalgran — COMMENTED** [review]
I am able to browse the drf-yasg and exercise endpoints without parameters but the parameter fields are currently all read-only for me.

**Comment by maurizi** [comment]
>When I browse the forms all of the fields are disabled/read-only. I tested in Firefox and Chrome and it appears to be a problem on all of the endpoints. Do you also see the fields as disabled?

There is a "Try it out" button that will enable the fields once clicked, and reveal an "Execute" button that will actually run the API request.

File: src/django/oar/urls.py [hunk 1]
```diff
@@ -77,13 +91,11 @@
...
 ]
 
-schema_view = get_swagger_view(title='Open Supply Hub API Documentation',
-                               patterns=public_apis)
-
 internal_apis = [
     url(r'^', include('django.contrib.auth.urls')),
     url(r'^web/environment\.js', environment, name='environment'),
-    url(r'^api/docs/', views.schema_view),
+    url(r'^api/docs/$', schema_view.with_ui('swagger', cache_timeout=0),
```
[the remark this answers is not in the document]
Comment (maurizi) [reply src/django/oar/urls.py:97]: Looking at the docs, it does look like `0` is already the default. Just copy/pasted from the MMW pull request w/o questioning if it was needed 😳

**Review by jwalgran — APPROVED** [review]
Thanks for explaining that "Try it out" unlocks the form editing. I was able to test the endpoints without issue 👍 

I compared the endpoints displayed on this branch to the existing staging site and verified that they are all match.

**Pull request merged**